ui/mainwindow.cpp

```diff
@@ -52,7 +52,9 @@
 #include <3rdparty/kuserfeedback/widgets/notificationpopup.h>
 #include <3rdparty/kuserfeedback/core/applicationversionsource.h>
 #include <3rdparty/kuserfeedback/core/compilerinfosource.h>
+#ifndef QT_NO_OPENGL
```

Comment:
remove, we can now always use openglinfosource again.

Reply:
Fixed